governance/pyth_staking_sdk/src/pyth-staking-client.ts

```diff
@@ -0,0 +1,204 @@
+        getConfigAddress()[0]
+      );
+    return convertBNToBigInt(globalConfigAnchor);
+  }
+
+  /** Gets a users stake accounts */
+  public async getStakeAccountPositions(
+    user: PublicKey
+  ): Promise<StakeAccountPositions[]> {
+    const res =
+      await this.stakingProgram.provider.connection.getProgramAccounts(
+        this.stakingProgram.programId,
+        {
+          encoding: "base64",
+          filters: [
+            {
+              memcmp: this.stakingProgram.coder.accounts.memcmp("positionData"),
+            },
+            {
+              memcmp: {
+                offset: 8,
+                bytes: user.toBase58(),
+              },
+            },
+          ],
+        }
+      );
+    return res.map((account) => {
+      const stakeAccountPositionsAnchor = new StakeAccountPositionsAnchor(
+        account.pubkey,
+        account.account.data,
+        this.stakingProgram.idl
+      );
+      return stakeAccountPositionsAnchor.toStakeAccountPositions();
+    });
+  }
+
+  public async getStakeAccountCustody(
+    stakeAccountPositions: PublicKey
+  ): Promise<Account> {
+    return getAccount(
+      this.connection,
+      getStakeAccountCustodyAddress(stakeAccountPositions)
+    );
+  }
+
+  public async initializePool({
+    rewardProgramAuthority,
+    poolData,
+    y,
+  }: {
+    rewardProgramAuthority: PublicKey;
+    poolData: PublicKey;
+    y: bigint;
+  }): Promise<void> {
+    const yAnchor = convertBigIntToBN(y);
+    const config = await this.getGlobalConfig();
+    await this.integrityPoolProgram.methods
+      .initializePool(rewardProgramAuthority, config.pythTokenMint, yAnchor)
+      .accounts({
+        poolData,
+      })
+      .rpc();
+  }
+
+  public async getOwnerPythATAAccount(): Promise<Account> {
+    const globalConfig = await this.getGlobalConfig();
+    return getAccount(
+      this.connection,
+      await getAssociatedTokenAddress(
+        globalConfig.pythTokenMint,
+        this.wallet.publicKey
+      )
+    );
+  }
+
+  public async getPoolConfigAccount(): Promise<PoolConfig> {
+    const poolConfigAnchor =
+      await this.integrityPoolProgram.account.poolConfig.fetch(
+        getPoolConfigAddress()
+      );
+    return convertBNToBigInt(poolConfigAnchor);
+  }
+
+  public async getPoolDataAccount(): Promise<PoolDataAccount> {
+    const poolConfig = await this.getPoolConfigAccount();
+    const poolDataAddress = poolConfig.poolData;
+    const poolDataAccountAnchor =
+      await this.integrityPoolProgram.account.poolData.fetch(poolDataAddress);
+    return convertBNToBigInt(poolDataAccountAnchor);
+  }
+
+  public async stakeToGovernance(
+    stakeAccountPositions: PublicKey,
+    amount: bigint
+  ) {
+    this.stakingProgram.methods
+      .createPosition(
+        {
+          voting: {},
+        },
+        new BN(amount.toString())
+      )
+      .accounts({
+        stakeAccountPositions,
+      })
+      .rpc();
```

Comment:
Transaction sending seems inconsistent. Here you use anchor's rpc and below you use `sendTransactions` from `solana-utils`

Reply:
fixed